models/issues/comment.go

```diff
@@ -279,8 +279,8 @@ type Comment struct {
 	DependentIssue   *Issue `xorm:"-"`
 
 	CommitID        int64
-	Line            int64 // - previous line / + proposed line
-	TreePath        string
+	Line            int64         // - previous line / + proposed line
+	TreePath        string        `xorm:"VARCHAR(1024)"`
```

Comment:
Would you like to do some research about:

* What's the max length of a git path
* What's the max limit of VARCHAR

?